### PR TITLE
Module 2: To Do list: interactive list

### PR DESCRIPTION
**Pull request by preciousakams**
### **Commit Details**

- Added clear completed task
- mark task as completed

### CODE REVIEW

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @kamzzy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request is descriptive 👏
## Required Changes ♻️

- [ ] Your App is working fine, but you missed important requirements:

- Please **add** a new JavaScript file and import it as a module. ([See project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md))

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/todo.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+const form = document.querySelector('.form');
```
Comment (Ismailco) [inline src/modules/todo.js:1]: - [ ] [optional] When modifying the task the changes don't update in the local storage, kindly try to fix the edit function

File: src/modules/todo.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+   
+  `;
+
+  if (item) {
+    list.replaceChild(node, item);
+  } else {
+    list.append(node);
+  }
+}
+function addTodo(description) {
```
Comment (Ismailco) [inline src/modules/todo.js:38]: - [ ] [optional] The index of the tasks is dos not start from 1 when adding a new task to the list, please consider fixing this so your app can work as expected

**Review by Ismailco — APPROVED** [review]
### Hi @kamzzy,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ All requirements meet
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/todo.js (hunk 2 above)
Comment (preciousakams) [reply src/modules/todo.js:38]: @Ismailco thanks

**Pull request merged**